input/docs/integrations/editors/rider/run-configurations.md

```diff
@@ -88,15 +88,29 @@ All settings are project - specific and stored in the `.idea` folder. To share s
 ### Generic settings
 
 * *Cake file extension*
-  This setting is used to find all Cake files and display them in the tool window.
+  This setting is used to find all Cake files and display them in the Cake tasks window.
   Default: `cake`
 * *Task Regex*
   This regular expression is used to parse tasks from the Cake files.
   Default: `Task\s*?\(\s*?"(.*?)"\s*?\)`
 * *Verbosity*
-  This is the default verbosity to use, when running a task directly from the tool window or when creating a new run configuration.
+  This is the default verbosity to use, when running a task directly from the Cake tasks window or when creating a new run configuration.
   Default: `normal`
 
+### Search paths
+
+Allows to modify search paths for building the [Cake tasks window](#running-builds-directly).
```

Comment:
Something seems wrong in this sentence (people are not building the Cake tasks window)